044_primenumberinarange.js

```diff
@@ -0,0 +1,14 @@
+let array = [];

```

Comment:
You do not need to declare array globally. Declare this locally inside the function.